packages/flutter_tools/lib/src/commands/analyze_once.dart

```diff
@@ -92,7 +92,8 @@ class AnalyzeOnce extends AnalyzeBase {
     });
 
     await server.start();
-    server.onExit.then((int exitCode) {
+    // Completing the future in the callback can't fail.
+    server.onExit.then((int exitCode) { // ignore: unawaited_futures
```

Comment:
Can the `server.onExit` future ever complete with an error?

Reply:
The comment was for the closure inside (in that if that closure has an error, it won't be handled by anyone (which is ok))